.github/CODEOWNERS

```
@@ -15,6 +15,7 @@
 /charts/prometheus-consul-exporter/ @gkarthiks @timm088
 /charts/prometheus-couchdb-exporter/ @gkarthiks
 /charts/prometheus-druid-exporter/ @iamabhishek-dubey
+/charts/prometheus-kafka-exporter/ @gkarthiks
```

@gkarthiks please add @golgoth31

```suggestion
/charts/prometheus-kafka-exporter/ @gkarthiks @golgoth31
```